portainer.json

```diff
@@ -0,0 +1,38 @@
+{
+  "Portainer": {
+    "website": "https://hub.docker.com/r/portainer/portainer/",
+    "version": "1.0",
+    "description": "Advanced Users Only!!!! Portainer is a lightweight advanced management UI which allows you to easily manage your Docker host or Swarm cluster.",

```

Comment:
As discussed in the related conversation, we can add more details and warnings on the caveats and important points to consider when using Portainer within Rockstor.

```
    "description": "<strong>Advanced Users Only!</strong> Use at your own risk.<p>Portainer is a lightweight advanced management UI which allows you to easily manage your Docker host or Swarm cluster.</p> <p>Based on the official Portainer docker image: <a href='https://hub.docker.com/r/portainer/portainer/' target='blank'>https://hub.docker.com/r/portainer/portainer/</a>",
    "more_info": "<h4>Documentation</h4><p>Please refer to Portainer's documentation for details on its functions: <a href='https://portainer.readthedocs.io/en/stable/' target='_blank'>https://portainer.readthedocs.io/en/stable/</a><h4>Important points for proper function with Rockstor</h4><p>Portainer includes a lot of features and brings new possiblities to manage your docker environment. Nevertheless, please note that it is running alonside Rockstor, meaning that some conflicts might arise. For instance, it is important to note that all volumes automatically created and managed by Portainer will reside inside the Rocstor share designed as the Rock-ons service's root. It is thus recommended to create shares from Rockstor's webUI first, and then mount them as volume when creating a new container from within Portainer.</p>",
```

Reply:
@phillxnet maybe a feature request for an Experimental or Beta flag would be good, I think a warning should suffice here though.